contentcuration/contentcuration/templates/policies/policy_accept.html

```diff
@@ -23,10 +23,12 @@ <h4>{% trans 'We have updated our policies. Please accept the following terms to
 </div>
 <script>
 	window.addEventListener('load', function(){
-		$("#id_accepted").on("change", function(event){
-			(event.target.checked)? $("#submit").removeAttr("disabled").removeClass("disabled") :
-				$("#submit").attr("disabled", true).addClass("disabled");
+		let acceptedCheckbox = document.getElementById('id_accepted');
+		let submitButton = document.getElementById('submit');
+		acceptedCheckbox.addEventListener('change', event => {
+			event.target.checked? submitButton.removeAttribute('disabled').removeClass('disabled') :
+				submitButton.setAttribute('disabled', 'disabled').addClass('disabled');
```

Comment:
The original logic was little obfuscated using a ternary without setting value, but also I don't think `setAttribute` and `removeAttribute` are chainable nor am I aware of `removeClass` or `addClass` methods on an HTML element.

Reply:
Nice catch!